### ISSUE TITLE
how its works?

### ISSUE THREAD

**Issue by Leom2004** [sl4v3k/Shapr3d_crack#33]
downloaded shapr3d and logged in, then started the latest patcher and then the program closed?

**Comment by sl4v3k** [comment]
check if exactly same version, if not, patcher will write data to wrong location and Shapr3d may crash

**Comment by sl4v3k** [comment]
to get console output, run patcher from Admin elevated cmd, check the output

**Comment by Leom2004** [comment]
the console is closing after running

**Comment by Leom2004** [comment]
how can i install the exact version?

**Comment by Leom2004** [comment]
ok i have the exact version of programm but how can i check it works?

**Comment by sl4v3k** [comment]
told You already, 
1. run cmd console as Administrator
2. cd to the dir where inmemory exe is placed (eg: cd c:\tmp\inmemory_patcher.exe
3. run inmemory_patcher.exe

**Comment by sl4v3k** [comment]
console is closing, this is normal behaviour if You run exe by double clicking on it

**Comment by Leom2004** [comment]
so it had worked could create more than 2 projects but after a restart i have to run this file again is that right?

**Comment by sl4v3k** [comment]
yes, as name says it is "in memory patcher" changes are not permanent

**Comment by LasTCK** [comment]
> evet, adından da anlaşılacağı gibi "hafızadaki yama" değişiklikleri kalıcı değildir

https://shapr3d.en.uptodown.com/windows
	Is there a cracked for this version?
Shapr3D
5.750.7750.0

**Comment by sl4v3k** [comment]
just pushed to git
